File: src/data/clean/clean_c_diabetes.R
# Pima Indians Diabetes Data Set

require(readr)

# Get working directory from command line arguments.
args <- commandArgs(trailingOnly = TRUE)
working_directory = args[1]

# Set working directory.
setwd("C:\\Users\\Marta\\Documents\\GitHub\\pythonic-learning-machine\\data_sets\\")

# Load raw data.
data_set <- read_csv("01_raw/c_diabetes/data_set.csv",
                     col_names = FALSE)

# Export cleaned data.
write.table(data_set, file = "02_cleaned//c_diabetes.csv", row.names = F, col.names = F, sep = ",")

File: src/data/clean/clean_c_diabetes.py
import os
import sys 
import pandas as pd 

#get working directory 
wd = os.getcwd() 
wd = os.path.realpath('..')
print(wd)
folder = os.path.join(wd, 'data_sets')
print(folder)
#wd = os.chdir(path)

#print(os.listdir())

data_set = os.path.join(folder, '01_raw', 'c_diabetes', 'data_set.csv')
c_diabetes = pd.read_csv(data_set, header=None)
print(c_diabetes.head())

outname = 'c_diabetes.csv'
outdir = '02_cleaned'
if not os.path.join(folder, outdir):
    os.mkdir(outdir)

fullname = os.path.join(folder, outdir, outname)
print(fullname)
#save data set to csv
c_diabetes.to_csv(fullname)